proxysql/build-binary-proxysql

```diff
@@ -12,7 +12,17 @@ set -o xtrace
 if [ -f /usr/bin/yum ]; then
     RHEL=$(rpm --eval %rhel)
     if [[ $RHEL -eq 8 ]]; then
-        sudo yum -y install cmake python2 gnutls-devel libtool || true
+        sudo yum -y install python2 gnutls-devel libtool || true
+        sudo yum remove -y cmake
+        sudo wget https://github.com/Kitware/CMake/releases/download/v3.19.8/cmake-3.19.8.tar.gz
+        sudo tar -zxvf cmake-3.19.8.tar.gz
+        cd cmake-3.19.8/
+        sudo ./bootstrap
+        sudo make
+        sudo make install
+        PATH=$PATH:/usr/local/bin
```

Comment:
Do we need this step? IIRC /usr/local/bin is already in the PATH variable

Reply:
Yes. This line is redundant. I have removed it.